tsl/test/expected/continuous_aggs_ddl-10.out

```diff
@@ -1048,3 +1045,36 @@ ORDER BY ranges;
  _timescaledb_internal._hyper_13_22_chunk | {"[20,40)"}
 (1 row)
 
+-- TEST drop chunks from continuous aggregates by specifying view name
+SELECT drop_chunks(
+    table_name => 'drop_chunks_view',
```

Comment:
Same here as above. What is the behavior of `cascade_to_materializations = true` when `drop_chunks` is called on the view?

Do we need to test?

Reply:
there are no dependencies on this view/materialized hypertable. But I will add a contrived case and test. This would help  when we extend functionality to create views based on existing views (like  1 day view based on 1 hour view).

Reply:
Update: contrived case doesn't work as we disable cont. aggregates based on existing cont. aggregates.